src/components/account-tab.js

```diff
@@ -56,18 +60,13 @@ function AccountTab({ formItemLayout, tailFormItemLayout }) {
   };
 
   const handleAccountFinish = ({ username, email }) => {
-    confirmIdentityWithPassword({
-      t,
-      username: state.username,
-      onOk: () => saveAccountData({ username, email }),
-      userApiClient
-    });
+    dialogs.confirmWithPassword(user.username, () => saveAccountData({ username, email }));
   };
 
   const handleResetPasswordClick = async () => {
     try {
-      await userApiClient.requestPasswordReset({ email: state.email });
-      message.success(t('passwordResetEmailSent', { email: state.email }));
+      await userApiClient.requestPasswordReset({ email: user.email });
```

Comment:
This means this was a bug before, if one would change the email, not save, then request password reset, it would be sent to the wrong email..

Reply:
Yes